### ISSUE TITLE
chrome.manifest issue

### ISSUE THREAD

**Issue by ghost** [tsolucio/cbThunderbird#1]
I think I've found a bug. Running build.sh and installing the resulting add-on, I can see the icon is missing and the 'open vtiger in tab' button is missing. Opening up the error console there is
```
Failed to load overlay from chrome://vtiger/content/vtiger.xul.
```
Extracting the XPI, I can see chrome.manifest has changed. Basically `jar:chrome/vttb3-0.512.jar!/` has been duplicated. So for example:
``` 
skin	vtiger	classic/1.0	jar:chrome/vttb3-0.512.jar!/skin/
```
has changed to
```
skin	vtiger	classic/1.0	jar:chrome/vttb3-0.512.jar!/jar:chrome/vttb3-0.512.jar!/skin/
```

Looks to be due to the following in build.sh
```
if [ -f "chrome.manifest" ]; then
  echo "Preprocessing chrome.manifest..."
  # You think this is scary?
  #s/^(content\s+\S*\s+)(\S*\/)$/\1jar:chrome\/$APP_NAME\.jar!\/\2/
  #s/^(skin|locale)(\s+\S*\s+\S*\s+)(.*\/)$/\1\2jar:chrome\/$APP_NAME\.jar!\/\3/
  #
  # Then try this! (Same, but with characters escaped for bash :)
  sed -i -r s/^\(content\\s+\\S*\\s+\)\(\\S*\\/\)$/\\1jar:chrome\\/$APP_NAME\\.jar!\\/\\2/ chrome.manifest
  sed -i -r s/^\(skin\|locale\)\(\\s+\\S*\\s+\\S*\\s+\)\(.*\\/\)$/\\1\\2jar:chrome\\/$APP_NAME\\.jar!\\/\\3/ chrome.manifest

  # (it simply adds jar:chrome/whatever.jar!/ at appropriate positions of chrome.manifest)
fi
```

Commenting out that bit or adjusting the chrome.manifest looks to fix it.

**Comment by ghost** [comment]
Fixed in #2

**Comment by joebordes** [comment]
Thanks!
